### PR TITLE
update VM readme and VM install docs to reference R interpreter adds

### PR DESCRIPTION
**Pull request by jeffsteinmetz**
### What is this PR for?
Updates Virtual Machine documentation to include SparkR information.

### What type of PR is it?
Documentation

### Questions:
* Does the licenses files need update?  No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by AhyoungRyu** [comment]
Looks good !

File: docs/install/virtual_machine.md [hunk 1]
```diff
@@ -175,10 +178,6 @@ plt.title('How fast do you want to go today?')
 show(plt)
 ``` 
 
+### R Extras
```
Comment (felixcheung) [inline docs/install/virtual_machine.md]: should there be a `id:r-extras` for the anchor link above `#r-extras` ?

File: docs/install/virtual_machine.md [hunk 2]
```diff
@@ -175,10 +178,6 @@ plt.title('How fast do you want to go today?')
...
 
-
-
-
-
-
-
+With zeppelin running, an R Tutorial notebook will be available.  The R packages required to run the examples and graphs in this tutorial notebook were installed by this virtual machine.
```
Comment (felixcheung) [inline docs/install/virtual_machine.md]: hmm, aren't we listing the packages like in Python?

File: docs/install/virtual_machine.md (hunk 1 above)
Comment (jeffsteinmetz) [inline docs/install/virtual_machine.md]: The issue is how github markdown works vs the doc generator.
Github (readme) markdown uses the name of the header, in lower case, with spaces converted to dashes.
Is there an example of how the doc md -> html compiler interprets anchors?  I looked around at all of the other zeppelin docs, and couldn't find any doc examples that used anchors to headers within the "current" document.  I can update for the docs once I confirm how the doc generator works.  Does it use `id`?  Any examples?

File: docs/install/virtual_machine.md (hunk 2 above)
Comment (jeffsteinmetz) [inline docs/install/virtual_machine.md]: yah, we could do that for sure.
The package names for Python are listed on line 96 (earlier), not toward the end where you commented.
We can include in the same location if that makes sense.

File: docs/install/virtual_machine.md (hunk 1 above)
Comment (felixcheung) [inline docs/install/virtual_machine.md]: ah, got it. hmm, I'm not sure - perhaps someone on https://jekyllrb.com/docs/permalinks/

Comment (jeffsteinmetz) [inline docs/install/virtual_machine.md]: Which pre-processor do we use for markdown (prior to Jekyll)?
I could dig around, but figured you might be able to point it out quickly.

**Comment by felixcheung** [comment]
LGTM

File: docs/install/virtual_machine.md (hunk 1 above)
Comment (felixcheung) [inline docs/install/virtual_machine.md]: It's just Jekyll, i guess - you can find the extensions here: https://github.com/apache/incubator-zeppelin/blob/master/docs/_config.yml

**Pull request closed without merging**